### PR TITLE
add me to authors

### PR DESCRIPTION
**Pull request by sylv256**
## Changelog
 - 

## Todo
 - [ ] Tested
 - [ ] Documentation Added

Fixes #

### CODE REVIEW

**Comment by IMS212** [comment]
absolute cringe

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
   "authors": [
     "hydos",
     "Mekal \"Valoeghese\" Covic",
-    "ramidzkh"
+    "ramidzkh",
+    "Tehc"
```
Comment (OroArmor) [inline src/main/resources/fabric.mod.json:12]: this is bloat

**Review by OroArmor — APPROVED** [review]
looks good. only minor changes needed

File: src/main/resources/fabric.mod.json (hunk 1 above)
Comment (valoeghese) [reply src/main/resources/fabric.mod.json:12]: Way too much bloat

**Review by valoeghese — APPROVED** [review]

**Comment by valoeghese** [comment]
Stop editing our comments

File: src/main/resources/fabric.mod.json (hunk 1 above)
Comment (valoeghese) [reply src/main/resources/fabric.mod.json:12]: Why can tehc edit the comments

**Comment by valoeghese** [comment]
Tehc I will delete your repo if you continue

**Comment by valoeghese** [comment]
I fixed the comments to what they were originally

**Pull request merged**